source/Xunit.Gherkin.Quick/CoreModel/Scenario/GherkinScenarioOutlineExtensions.cs

```diff
@@ -12,7 +12,7 @@ namespace Xunit.Gherkin.Quick
 {
     internal static class GherkinScenarioOutlineExtensions
     {
-        private static readonly Regex _placeholderRegex = new Regex(@"<([^>]+)>");
+        private static readonly Regex _placeholderRegex = new Regex(@"<([a-zA-Z0-9]([^<>]*[a-zA-Z0-9])?)>");
```

Comment:
I think the outer parentheses are redundant. It must work without them as well, so please remove them if so.

Reply:
I thought so as well, but it starts catching empty tokens without them:
`Examples table did not provide value for ```